docs/changes.xml

```diff
@@ -97,13 +187,23 @@ ServerRequest.httpVersion variable format in Node.js module.
 </para>
 </change>
 
+<change type="bugfix">
+<para>
+Node.js module name rules, notable for Nuxt applications.
```

Comment:
```suggestion
Node.js module now handles standard library imports prefixed with "node:", making it possible to run newer Nuxt applications, among others.
```

Reply:
the word "now" seems redundant here and elsewhere. I would prefer to get rid of it wherever it already exists, instead of spreading it. All other changes look good, I'll add them in a few hours. Thanks!